operator/cmd/root.go

```diff
@@ -276,6 +277,8 @@ var (
 			// configuration to describe, in form of prometheus metrics, which
 			// features are enabled on the operator.
 			features.Cell,
+
+			featuresNoop.Cell,
```

Comment:
One of the two variants of the metrics should be provided in kvstoremesh as well: https://github.com/cilium/cilium/blob/pr%2Faanm%2Ffeatures-3/clustermesh-apiserver/kvstoremesh/cells.go#L22-L55

Just a note that providing the noop implementation in kvstoremesh would potentially mean having a partial view of the connections from the local cluster when kvstoremesh is enabled, as all the agents would report kvstoremesh, and it would not be possible to distinguish whether the remote clusters actually use the clustermesh-apiserver or an etcd cluster (assuming that's of interest here).